zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -1889,6 +1902,13 @@ impl<F: Field> RlpCircuitConfig<F> {
                 for i in sm_rows.len()..last_row {
                     self.assign_sm_end_row(&mut region, i)?;
                 }
+                region.assign_fixed(|| "q_first", self.q_first, 0, || Value::known(F::one()))?;
+                region.assign_fixed(
+                    || "q_last",
+                    self.q_last,
+                    last_row,
+                    || Value::known(F::one()),
+                )?;
```

Comment:
`last_row` is the circuit size (`self.size`). So we should in fact use `last_row - 1` as the offset for marking `q_last == 1`. Right?

Reply:
yeah, you're right.